proposed/extended-coding-style-guide.md

```diff
@@ -122,22 +125,77 @@ declaration.
 
 There MUST be one `use` keyword per declaration.
 
-There MUST be one blank line after the `use` block.
+When using multiple classes, functions or constants within one namespace, you
+MUST group use statements within one namespace.
+
+Use statements MUST be in a block according to the entity (class, function or group)
```

Comment:
entity means different type of thing. So class is an entity, function is an entity and constant is an entity. So basically classes/interfaces are one block, functions are another block and constants are another block. But all constants must be in the constant block, all functions must be in the function block etcetera. Does that make sense?